packages/synapse-react-client/src/components/GenericCard/GenericCard.tsx

```diff
@@ -716,3 +723,15 @@ export default class GenericCard extends React.Component<
     )
   }
 }
+
+export default function GenericCard(props: GenericCardProps) {
+  const { entity: table } = useQueryContext()
+  const queryVisualizationContext = useQueryVisualizationContext()
+  return (
+    <_GenericCard
+      {...props}
+      table={table}
+      queryVisualizationContext={queryVisualizationContext}
+    />
+  )
+}
```

Comment:
Just added a function component wrapper that can get the context values and pass into the internal component.

This pattern will also work for the next state management solution coming in a later PR.